examples/miral-shell/miral-screencast.sh

```diff
@@ -45,4 +45,5 @@ trap 'kill ${mirscreencast_pid}; rm -f -- "${tempfile}"; exit 0' INT TERM HUP EX
 sleep 1; # don't lose the next message in the spew from mirscreencast
 read -rsp $'\n\nPress enter when recording complete...'
 
+kill ${mirscreencast_pid}
```

Comment:
Will this not play bad with the `trap` above? Maybe it should be `kill -SIGKILL || /bin/true` there?